KyleHouston/Problem1.js

```diff
@@ -0,0 +1,146 @@
+function makeCard(id) {
```

Comment:
Looks good! It seems like you didn't get to 2e (the protected `push` method). No worries there, but if you happen to stumble across any free time, that might be worth checking out.